MANIFEST.in

```diff
@@ -3,6 +3,7 @@ graft tests
 graft requirements
 
 include AUTHORS.rst LICENSE*.rst README.rst
+include docs/examples/data/KFe2As2-00838.tif
```

Comment:
please do a recursive include of all the examples.  People will need all the examples in general.  Probably graft docs/examples?